src/query/config/src/config.rs

```diff
@@ -1214,6 +1214,15 @@ pub struct QueryConfig {
 
     #[clap(long, default_value = "10000")]
     pub max_query_log_size: usize,
+    /// Parquet file with smaller size will be read as a whole file, instead of column by column.
+    /// For example:
+    /// parquet_fast_read_bytes = 52428800
+    /// will let databend read whole file for parquet file less than 50MB and read column by column
+    /// if file size is greater than 50MB
+    pub parquet_fast_read_bytes: Option<u64>,
```

Comment:
missed a #[clap] header here?

Reply:
if 52428800 is a feasible default config for the most workloads, we can add a default_value here?

Reply:
I think it depends on the storage io bandwidth for exampl this setup only take significant value for R2, otherwise it seems cannot introduce significant changes.

Reply:
IMO this parameter is benefical for the common workload, not only for the r2 workload.

@youngsofun @BohuTANG Is this understanding correct?